### ISSUE TITLE
browser provider not found if both testcafe and testcafe-browser-provider-browserstack installed globally

### ISSUE THREAD

**Issue by dschuldt** [DevExpress/testcafe-browser-provider-browserstack#71]
Hi,

if both testcafe and testcafe-browser-provider-browserstack are installed with -g flag, the command

`testcafe -b browserstack `

fails with

> ERROR The specified "browserstack" browser provider was not found.

This error can be reproduced with Ubuntu (18.04.2 LTS (Bionic Beaver)") and Node v8.16.0 as well as the latest Docker Image, with the following Dockerfile

```
FROM testcafe/testcafe:1.1.3
User root
RUN npm install -g testcafe-reporter-html testcafe-browser-provider-browserstack
```

Thank you for your help

Regards

**Comment by miherlosev** [comment]
Hi @dschuldt 

It's well-know restriction inspired by the node's resolution modules [algorithm](https://nodejs.org/api/modules.html#modules_all_together).
To resolve this situation, you need to specify the path in  `NODE_PATH` environment variable as described in [nodejs documentation](https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders)

**Comment by dschuldt** [comment]
Thanks a lot @miherlosev 
Can you please elaborate your answer; where to specify the peace of information exactly?

Adding `ENV NODE_PATH /usr/lib/node_modules` to the Dockerfile is unsufficient apparently

**Comment by yashLadha** [comment]
Not working @miherlosev tried setting to `npm root -g` but didn't worked out.

**Comment by AlexKamaev** [comment]
@yashLadha
This is a command that shows the path to the global `node_modules` folder.
Please execute this command, get the shown path and check if the `testcafe` and `testcafe-browser-provider-browserstack` directories exists.
Please also check that you do not have testcafe installed locally, because in this case, the local testcafe will be executed, and the global `testcafe-browser-provider-browserstack` will not be found.

**Comment by yashLadha** [comment]
I have both testcafe and the provider installed but still, testcafe is unable to find the provider. Investigated the code about how this provider is resolved and founded that it is doing a `require` statement in testcafe. It shouldn't fail in that case. Can you please look into this issue.

**Comment by AndreyBelym** [comment]
Thank you for your report. I've reproduced the issue, it was caused by an incorrect dependency in `testcafe-browser-provider-browserstack`. It is fixed in version `1.10.1`. I have also submitted a suggestion about not hiding the original `require` error to the main TestCafe repository: DevExpress/testcafe#4522.
